### PR TITLE
Add Luxembourg

### PR DESCRIPTION
**Pull request by ivorbosloper**
Solve https://github.com/fiboa/data/issues/24

### CODE REVIEW

File: fiboa_cli/datasets/lu.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        "roles": ["licensor", "distributor"]
+    }
+]
+ATTRIBUTION = "Luxembourg ministry of Agriculture"
+LICENSE = {
+    "title": "This layer is published under the terms of the license Creative Commons Attribution 4.0 International (CC BY 4.0)",
+    "href": "https://creativecommons.org/licenses/by/4.0/",
+    "type": "text/html",
+    "rel": "license"
+}
```
Comment (m-mohr) [inline fiboa_cli/datasets/lu.py:36]: This can be simplified.

```suggestion
LICENSE = "CC-BY-4.0"
```

File: fiboa_cli/datasets/lu.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+COLUMNS = {
+    "geometry": "geometry",
+    "FLIK": "id",
```
Comment (m-mohr) [inline fiboa_cli/datasets/lu.py:40]:
```suggestion
    'FLIK': ['id', 'flik'], # make flik id a dedicated column to align with NRW etc.
```

File: fiboa_cli/datasets/lu.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+]
+ATTRIBUTION = "Luxembourg ministry of Agriculture"
+LICENSE = {
+    "title": "This layer is published under the terms of the license Creative Commons Attribution 4.0 International (CC BY 4.0)",
+    "href": "https://creativecommons.org/licenses/by/4.0/",
+    "type": "text/html",
+    "rel": "license"
+}
+
+COLUMNS = {
```
Comment (m-mohr) [inline fiboa_cli/datasets/lu.py:38]:
```suggestion
EXTENSIONS = [
    "https://fiboa.github.io/flik-extension/v0.1.0/schema.yaml"
]
COLUMNS = {
```

File: fiboa_cli/datasets/lu.py [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    convert_(
+        output_file,
+        cache,
+        SOURCES,
+        COLUMNS,
+        ID,
+        TITLE,
+        DESCRIPTION,
+        providers=PROVIDERS,
+        attribution=ATTRIBUTION,
```
Comment (m-mohr) [inline fiboa_cli/datasets/lu.py:55]:
```suggestion
        extensions=EXTENSIONS,
        attribution=ATTRIBUTION,
```

**Review by ivorbosloper — COMMENTED** [review]
Thanks, good suggestion @m-mohr

File: fiboa_cli/datasets/lu.py [hunk 5]
```diff
@@ -0,0 +1,57 @@
+import os
+
+from ..convert_gml import gml_assure_columns
+from ..util import log
+import geopandas
+
+from ..convert_utils import convert as convert_
```
Comment (m-mohr) [inline fiboa_cli/datasets/lu.py:7]:
```suggestion
from ..convert_utils import convert as convert_
```

None of the imports is actually used, right?

**Review by m-mohr — APPROVED** [review]

File: fiboa_cli/datasets/lu.py (hunk 5 above)
Comment (ivorbosloper) [reply fiboa_cli/datasets/lu.py]: Good point. I currently don't run an import-optimizer. If we could pick a code style with a deterministic code formatter (ruff, black, pylint) we could run that automatically before every commit (with a git hook) and check code style with github action. Just fixed it now... :)

**Comment by ivorbosloper** [comment]
Hmmm. Too soon I'm afraid. I tried re-generating + publishing the Luxembourg files for source.coop to include the FLIK attribute, but the FLIK format is not the same. Should we revert or should the FLIK spec be more flexible?

```
Validating fiboa/data/lu/luxembourg.parquet
  - fiboa version: 0.2.0
    fiboa extensions:
      - https://fiboa.github.io/flik-extension/v0.1.0/schema.yaml
  - flik: String '0170586' is shorter than the minimum length of 16.
  - flik: String '0170586' does not match the required pattern: ^[A-Z]{2}[A-Z0-9]{2}[A-Z0-9]{2}[A-Z0-9]{10}$.
  ```

**Comment by m-mohr** [comment]
That's not a valid FLIK ID. Maybe they removed the prefix? Let me check... 🤔

**Pull request merged**